### PR TITLE
BZ 1542912 Change hostnames option when creating new registry certificates

### PR DESCRIPTION
**Pull request by ghost**
https://bugzilla.redhat.com/show_bug.cgi?id=1542912

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL.

File: install_config/redeploying_certificates.adoc [hunk 1]
```diff
@@ -533,14 +533,14 @@ $ REGISTRY_HOSTNAME=`oc get route/docker-registry -o jsonpath='{.spec.host}'`
...
-    --hostnames=$REGISTRY_IP,docker-registry.default.svc.cluster.local,$REGISTRY_HOSTNAME \
+    --hostnames=$REGISTRY_IP,docker-registry.default.svc,docker-registry.default.svc.cluster.local,$REGISTRY_HOSTNAME
     --cert=/etc/origin/master/registry.crt \
     --key=/etc/origin/master/registry.key \
     --signer-serial=/etc/origin/master/ca.serial.txt
 ----
 +
 Run `oc adm` commands only from the first master listed in the Ansible host inventory file,
-by default *_/etc/ansible/hosts_*.
+by default *_/etc/ansible/hosts_*. See the link:https://github.com/openshift/openshift-ansible/blob/release-3.7/playbooks/common/openshift-cluster/redeploy-certificates/registry.yml#L70[Ansible playbook] when redeploying registry certificates. If `docker-registry.default.svc` is not added, the new generated certificate does not have `docker-registry.default.svc` DNS in the SAN.
```
Comment (mburke5678) [inline install_config/redeploying_certificates.adoc]: Do we need the link here? It is a repeat of your code example and doesn't add much (for me, anyway).

I am not sure what this mean: "`docker-registry.default.svc` DNS in the SAN" I see it in the BZ, so maybe it is OK.

**Comment by mburke5678** [comment]
@tmorriso-rh Couple of questions. Otherwise LGTM

File: install_config/redeploying_certificates.adoc (hunk 1 above)
Comment (gpei) [reply install_config/redeploying_certificates.adoc]: We don't need such explanation in an official doc IMO

**Comment by tmorriso-rh** [comment]
@mburke5678 Thanks. Changes made. PTAL. I also changed two instances of "OpenShift" to the {product-title} attribute.

**Comment by vikram-redhat** [comment]
@tmorriso-rh  - also this one. Did you want to go ahead and merge this one and CP to branches, then tag me before merging?

**Comment by vikram-redhat** [comment]
@tmorriso-rh - should be marked with next release as it is going into the 3.6 branch onwards.

**Pull request merged**